### PR TITLE
Abdullah 9.9

### PR DESCRIPTION
**Pull request by abdullahtariq**

### CODE REVIEW

File: apps/bridgemail/contacts/subscriber.js [hunk 1]
```diff
@@ -109,7 +113,7 @@ define(['text!contacts/html/subscriber.html', "contacts/subscriber_timeline","co
                     else{
                         URL = "/pms/io/subscriber/getData/?BMS_REQ_TK=" + bms_token + "&sfid=" + this.sub_id + "&type=getSubscriberBySfInfo&email="+this.email;
                     }
-                    jQuery.getJSON(URL, function(tsv, state, xhr) {
+                    this.getSubscriber = jQuery.getJSON(URL, function(tsv, state, xhr) {
```
Comment (umairmakesbridge) [inline apps/bridgemail/contacts/subscriber.js]: make this a local variable

File: apps/bridgemail/contacts/subscriber.js [hunk 2]
```diff
@@ -163,18 +167,33 @@ define(['text!contacts/html/subscriber.html', "contacts/subscriber_timeline","co
...
+                }
+                      
+            },
                 getActiviites: function(){
                   //Loading subscriber activities like last seen, visists and actions 
                     var _this = this;
                     this.app.showLoading("States..", this.$(".sub-stats"));
                     URL = "/pms/io/subscriber/getData/?BMS_REQ_TK=" + bms_token + "&subNum=" + this.sub_id + "&type=getActivityStats";
-                    jQuery.getJSON(URL, function(tsv, state, xhr) {
+                   this.getTimeLine = jQuery.getJSON(URL, function(tsv, state, xhr) {
```
Comment (umairmakesbridge) [inline apps/bridgemail/contacts/subscriber.js]: also make this a single variable.

File: apps/bridgemail/contacts/contacts.js [hunk 3]
```diff
@@ -323,9 +339,9 @@ function (subscriberCollection,template,SubscriberRowView,addContactView) {
                 });
                 // add into enqueueAjax Request
                 if(this.$el.parents('body').find('#wstabs li.active').attr('workspace_id')){
-                    this.app.enqueueAjaxReq.push(this.contacts_request); 
+                    this.enqueueAjaxReq.push(this.contacts_request); 
                 }      
-                console.log(this.app.enqueueAjaxReq);
```
Comment (umairmakesbridge) [inline apps/bridgemail/contacts/contacts.js]: remove console logs from all places

**Review by umairmakesbridge — CHANGES_REQUESTED** [review]

File: apps/bridgemail/contacts/subscriber.js (hunk 1 above)
Comment (abdullahtariq) [reply apps/bridgemail/contacts/subscriber.js]: Applied as request

**Pull request merged**